airflow/api_connexion/schemas/task_schema.py

```diff
@@ -26,14 +26,16 @@
     TimeDeltaSchema,
     WeightRuleField,
 )
-from airflow.api_connexion.schemas.dag_schema import DAGSchema
 from airflow.models.mappedoperator import MappedOperator
 
 if TYPE_CHECKING:
     from airflow.models.operator import Operator
 
+class SubDagMixin:
+    from airflow.api_connexion.schemas.dag_schema import DAGSchema
```

Comment:
What is the purpose of move import inside of class definition?

Reply:
As I was getting cyclic dependency issues earlier that's why I moved the import inside the function. We can move this import outside of class definition.